app/models/configuration.rb

```diff
@@ -55,4 +55,8 @@ module Configuration
   # for subscriptions. See app/controllers/subscriptions_controller.rb
   STRIPE_PUBLISHABLE_KEY = 'xxxxxxxxxxxxxxxxx'
   STRIPE_SECRET_KEY = 'xxxxxxxxxxxxxxxxx'
+
+  DEVISE_SECRET_KEY = "9c92ee433b851b3db3952e81e0c9d678de778b194986e2ae8b160eca08bea4696152801e83d88b75a17375867b07e46d70d9bfb7ff5a352c2e1de8790448eaa5"
+
+  SECRET_KEY_BASE = "39d782b099a06652f92585de84f568c71885a4a129a307f03d2a803c6298ea129e334442ab042dc7fa664adc6e28b41317c3d53291366d0d7b5761c65102f3d4"
```

Comment:
Yes. In the nasty way we have configuration set up in production this file is overwritten completely with the real values: https://github.com/openaustralia/planningalerts/blob/master/config/deploy.rb#L47

Why use what look like real values, you ask? Well we had them committed before so I thought, meh, why not keep the same keys in development (e.g. https://github.com/openaustralia/planningalerts/pull/671/files#diff-afe98573d9aad940bb0f531ea55734f8L7 )